omise/classes/charge.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
@guzzilar Are there other parts that need to be changed?

To merge the pull request, Is it waiting for only DocBlocks update (https://github.com/omise/omise-prestashop/pull/19#discussion_r102140277)?